codingtovictory/day6/takeHomeChallenge/style.css

```diff
@@ -0,0 +1,46 @@
+.FirstA {   
+    font-size: 25px;
+    padding-left: 20px;
+
+}
+
+.SecondA { 
+    font-size: 25px;
+    padding-left: 20px;
+    margin-top: 20px;
+
+}
+
+.ThirdA { 
+    font-size: 25px;
+    padding-left: 20px;
+    margin-top: 20px;
+
+}
```

Comment:
Since all of these classes have the same styling I would suggest giving them all one class name and applying the styling. There would be less code and more efficient, ex.:
```
.links{
    font-size: 25px;
    padding-left: 20px;
    margin-top: 20px;
}
```
And then you could give an id to the first link and give a margin top of 0